### PR TITLE
Add link to pyramid-oauthlib

### PR DESCRIPTION
**Pull request by tilgovi**
Disclosure: I am the author of pyramid-oauthlib

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -55,6 +55,7 @@ The following packages provide OAuth support using OAuthLib.
 
 - For Django there is `django-oauth-toolkit`_, which includes `Django REST framework`_ support.
 - For Flask there is `flask-oauthlib`_ and `Flask-Dance`_.
+- For Pyramid there is `pyramid-oauthlib`.
```
Comment (wiliamsouza) [inline README.rst]: Missing one `_` after `pyramid-oauthlib`.

**Comment by tilgovi** [comment]
Fixed! Thanks.

**Comment by wiliamsouza** [comment]
:+1: LGTM PR #333

**Comment by tilgovi** [comment]
I think Travis might be failing because #322 needs to be merged (pyjwt is not pinned to any particular version).

File: README.rst [hunk 2]
```diff
@@ -55,6 +55,7 @@ The following packages provide OAuth support using OAuthLib.
 
 - For Django there is `django-oauth-toolkit`_, which includes `Django REST framework`_ support.
 - For Flask there is `flask-oauthlib`_ and `Flask-Dance`_.
+- For Pyramid there is `pyramid-oauthlib_`.
```
Comment (ib-lundgren) [inline README.rst]: `pyramid-oauthlib_` should be `pyramid-oauthlib`_ :)

**Comment by tilgovi** [comment]
<sigh> ... syntax.

Fixed!

**Pull request merged**